reflection.md

```diff
@@ -1,9 +1,11 @@
-The goal of our redesigned Picture library was to make it easer to compose
+The goal of our redesigned Picture library was to make it easier to compose
 operations without having to load and save an image each time. We accomplished
 that goal, now it's time for critique. In this round of critique we're focusing
 on the following question:
 
 Is there anything that is still annoying or difficult about the operations we
 have in the library (i.e., flipping, rotating, and grayscaling)?
 
-_replace this text with your answer_
+I was trying to define a helper function to skip the duplicate part. However, it did not work out because I couldn't
+make a input into the argument such as (width - column - 1, row). Also I found annoy the output showed up as "Unit" and
+I do not what is going on.
```

Comment:
Yes, a lot of the `Picture` library implementation is repeated, and could be simplified. As you mentioned, it's not easy to see how to simplify it. We may see some techniques in the future that will help, though...

Also, could you say more about the output showing up as `Unit`? Which output do you mean?

Reply:
This question was also intended to get at the _user's_ perspective: From the user's point of view (e.g,. when writing `Practice.scala`, is there anything still annoying about the library?

Reply:
"def flipHorizontal(bufferedImage_name: BufferedImage): BufferedImage = {"
So for the above code structure, when I first implemented it, it said that the returned value was "Unit" instead of "BufferedImage." Then later I googled and figured out that "Unit" meant empty return value. And I re-check my work, found there was an error and solved that problem.

From the user's point of view, I thought the Scala should make things easier like having all of the argument together instead of calling again and again. And we talked about how to make these happen in our session so problem solved!